css/style.css

```diff
@@ -37,6 +37,12 @@ canvas {
     left: 0px;
 }
 
+.shadow {
```

Comment:
Since we're editing the CSS and JS files. do we want to change the version number in the html file? So the browser knows to clear cache

Reply:
true. on it